samples/packages/openapi-spec-loader/src/main/kotlin/OpenAPISpecLoaderCfg.kt

```diff
@@ -16,8 +16,10 @@ import javax.annotation.processing.Generated
 @JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
 data class OpenAPISpecLoaderCfg(
     @JsonProperty("import_type") val importType: String = "URL",
+    @JsonProperty("import_style") val importStyle: String = "single",
     @JsonProperty("spec_url") val specUrl: String = "",
     @JsonProperty("spec_file") val specFile: String = "",
+    @JsonProperty("directory") val directory: String = "",
```

Comment:
I think these could be confusing — easy for folks to make mistakes in how they set these up (?) What I had in mind was rather automatically detecting if multiple files could be involved:

- Direct upload of a .zip file
- Object store details that only specifies the prefix in the object store, but no object key

Any other scenario (direct upload of a JSON file, or object store details that includes the object key) would handle just a single file as input.